docs/WritingTests.asciidoc

```diff
@@ -246,7 +250,8 @@ clone_job.pl --from https://openqa.opensuse.org  --host localhost 24 TESTDEBUG=1
 ----
 
 * Create a job again, this time setting the +SKIPTO+ variable to the snapshot
-which failed on previous run. If clone_job script is not used, +TESTDEBUG=1+
+which failed on previous run. If the clone_job script is not used on the job
```

Comment:
sorry but it's not get more clear for me, suggesting "Make sure that this new job stil have TESTDEBUG=1"